doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -949,16 +949,25 @@ Interpretation scopes can include an interpretation for numerals and
 strings. However, this is only made possible at the Objective Caml
 level.
 
-See :ref:`above <NotationSyntax>` for the syntax of notations including the
-possibility to declare them in a given scope. Here is a typical example which
-declares the notation for conjunction in the scope ``type_scope``.
+.. cmd:: Declare Scope @scope
+
+   This adds a new scope named :n:`@scope`. Note that the initial
+   state of Coq declares by default the following interpretation scopes:
+   ``core_scope``, ``type_scope``, ``function_scope``, ``nat_scope``,
+   ``bool_scope``, ``list_scope``, ``int_scope``, ``uint_scope``.
```

Comment:
Are these built-in scopes or scopes declared in the prelude?

Reply:
All names are declared in the prelude, but some have a specific built-in action corresponding to the class they are bound to (e.g. `CL_SORT` <-> `type_scope`, and `CL_FUN` <-> `function_scope`).